Assets/MRTK/SDK/Features/UX/Scripts/UnityUI.meta

```diff
@@ -1,5 +1,5 @@
 fileFormatVersion: 2
```

Comment:
Should the scripts that were moved have their namespaces updated? Having an SDK script in the `Examples.Demos` namespace feels odd (commenting here since i can't comment directly on the scripts)

Reply:
most likely. looks like we missed that in the other PR

Reply:
fixed by #8780